.circleci/config.yml

```diff
@@ -62,6 +62,7 @@ jobs:
               command: |
                   source ~/miniconda/bin/activate testenv
                   cd doc/ && make html
+              no_output_timeout: 20m
```

Comment:
I think it'd be best to keep this at 10 min if it's not the actual issue. In general we're trying to avoid having our documentation take a long time to build until we have a clear plan in place for something like the GUI tutorials where long build times are a bit unavoidable.

Reply:
Ok sounds good